keras_cv/utils/preprocessing.py

```diff
@@ -14,6 +14,8 @@
 import tensorflow as tf
 from tensorflow.keras import backend
 
+import keras_cv.bounding_box.converters as bbox_formater
```

Comment:
we don't need the bbox_formatter import.  You can just use `bounding_box.convert_format()`.  prefer top level namespaces when possible